### PR TITLE
Added custom mock results

### PR DESCRIPTION
**Pull request by tarzzz**
This PR covers some custom mock results for queries, which are used in testing Falcon against Workspace `sql_panel`.

### CODE REVIEW

**Comment by tarzzz** [comment]
@n-riesco Ready for a review.
cc: @scjody

**Comment by tarzzz** [comment]
To test this:

  - Go to: https://local.plot.ly/create/?upload=sql&url=https://plotly-connector-pr-336.herokuapp.com/
  - Connect to the Falcon, and run the queries in `mockResults` objects. The results should reflect the values in `mockResults`.

File: test/backend/datastores.mock.spec.js [hunk 1]
```diff
@@ -79,6 +80,26 @@ describe('Datastore Mock:', function () {
         });
     });
 
+    it('custom queries return custom results', function() {
+        const promises = [];
+        let promise = null;
+
+        // Check each case iteratively:
+        map(key => {
+          promise = POST(`connections/${connectionId}/query`, {query: key})
```
Comment (n-riesco) [inline test/backend/datastores.mock.spec.js]: 4-space indent

**Comment by n-riesco** [comment]
:dancer: after fixing the indent.

**Pull request merged**